lincheck/src/test/java/org/jetbrains/kotlinx/lincheck/test/verifier/linearizability/ClocksTest.kt

```diff
@@ -0,0 +1,103 @@
+/*-
+ * #%L
+ * Lincheck
+ * %%
+ * Copyright (C) 2019 - 2020 JetBrains s.r.o.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.jetbrains.kotlinx.lincheck.test.verifier.linearizability
+
+import org.jetbrains.kotlinx.lincheck.*
+import org.jetbrains.kotlinx.lincheck.annotations.*
+import org.jetbrains.kotlinx.lincheck.execution.*
+import org.jetbrains.kotlinx.lincheck.strategy.stress.*
+import org.junit.*
+import kotlin.reflect.jvm.*
+
+
+@StressCTest(generator = ClocksTestScenarioGenerator::class, iterations = 1,
+             actorsBefore = 0, threads = 2, actorsPerThread = 1, actorsAfter = 0,
+             sequentialSpecification = ClocksTestSequential::class,
+             requireStateEquivalenceImplCheck = false, minimizeFailedScenario = false)
```

Comment:
We were going to add some error message referring to check the `minimizeFailedScenario` parameter in case of the hanging execution like here

Reply:
We do not have such issues in practice (only in our internal tests, and only once so fat) so that such a message seems redundant. Let's just keep this in mind.